test/interpreter/function-covariance.s.spec.ts

```diff
@@ -0,0 +1,92 @@
+import {HyperFormula} from '../../src'
+import {ErrorType} from '../../src/Cell'
+import {ErrorMessage} from '../../src/error-message'
+import {adr, detailedError} from '../testUtils'
+
+describe('COVARIANCE.S', () => {
+  it('validates number of arguments',  () => {
+    const engine =  HyperFormula.buildFromArray([
+      ['=COVARIANCE.S(B1:B5)'],
+      ['=COVARIANCE.S(B1:B5, C1:C5, D1:D5)'],
+    ])
+
+    expect(engine.getCellValue(adr('A1'))).toEqualError(detailedError(ErrorType.NA, ErrorMessage.WrongArgNumber))
+    expect(engine.getCellValue(adr('A2'))).toEqualError(detailedError(ErrorType.NA, ErrorMessage.WrongArgNumber))
+  })
+
+  it('ranges need to have same amount of elements',  () => {
+    const engine =  HyperFormula.buildFromArray([
+      ['=COVARIANCE.S(B1:B5, C1:C6)'],
+    ])
+
+    expect(engine.getCellValue(adr('A1'))).toEqualError(detailedError(ErrorType.NA, ErrorMessage.EqualLength))
```

Comment:
There will be different error if they're empty or have a single arg? "If either array1 or array2 is empty or contains only 1 data point each, COVARIANCE.S returns the #DIV/0! error value."

https://support.microsoft.com/en-us/office/covariance-s-function-0a539b74-7371-42aa-a18f-1f5320314977

Reply:
we have test for this